src/services/mapService.tsx

```diff
@@ -57,7 +57,6 @@ export async function fetchRegionDatetimes(aoiId: AoiId): Promise<number[]> {
         console.error('Error loading region datetimes:', error)
         throw error
     }
```

Comment:
Line 29:  // todo use tanstack query

I would either make a ticket and do it or remove the comment and live without the tanstack query. Todo-comments in the code are always forgotten about and don't look nice, I think